checkstyle-checks/src/main/resources/com/sonatype/checks/packagenames.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+
+    Copyright (c) 2011-present Sonatype, Inc. All rights reserved.
+    Includes the third-party code listed at http://links.sonatype.com/products/clm/attributions.
+    "Sonatype" is a trademark of Sonatype, Inc.
+
+-->
+<!DOCTYPE checkstyle-packages PUBLIC
+    "-//Puppy Crawl//DTD Package Names 1.0//EN"
+    "http://www.puppycrawl.com/dtds/packages_1_0.dtd">
+
+<checkstyle-packages>
+  <package name="com.sonatype.checks"/>
+  <package name="com.puppycrawl.tools.checkstyle">
```

Comment:
If I'm understanding the checkstyle docs and the intent of this file correctly, you don't need to specify the default checkstyle modules in this file.  The note at the bottom of [this](http://checkstyle.sourceforge.net/config.html#Packages) section seems to say so.

Reply:
> If I'm understanding the checkstyle docs and the intent of this file correctly, you don't need to specify the default checkstyle modules in this file.

Turns out we don't need the file at all unless we plan on using short names for checks, which I deliberately did not do so that it is easy to recognize which checks specified in the checkstyle config are Sonatype custom checks.